internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -352,7 +352,17 @@ func (u *ALMExamplesUpdate) Apply(csv *olmapiv1alpha1.ClusterServiceVersion) err
 		}
 	}
 	sb.WriteString(`]`)
-
-	csv.GetAnnotations()["alm-examples"] = sb.String()
+	examplesJSON, err := prettyJSON([]byte(sb.String()))
+	if err != nil{
```

Comment:
Another minor nit (space between `nil` and `{`). This is based on Go's standard code formatting. Most text editors and IDEs can run this for you every time you save files. See https://blog.golang.org/go-fmt-your-code

```suggestion
	if err != nil {
```

Reply:
@camilamacedo86 make sure you run `make format` from the project root.

Reply:
👍